### PR TITLE
feature: github issue forms

### PR DESCRIPTION
**Pull request by leonsteinhaeuser**
This PR removes Markdown issue templates and adds GitHub issue forms.

Closes: #48

### CODE REVIEW

**Review by scarf005 — CHANGES_REQUESTED** [review]
Very nice! however adding `questions` issue might be confusing over `Q&A` in discussions tab. i'd suggest adding a link to guide them to discussions.

File: .github/ISSUE_TEMPLATE/question.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    id: question
+    attributes:
+      label: Describe the question
+      description: A clear and concise description of what the question is about.
```
[the remark this answers is not in the document]
Comment (leonsteinhaeuser) [reply .github/ISSUE_TEMPLATE/question.yml:22]: Good point. I will add your change.

**Review by scarf005 — APPROVED** [review]
great!
maybe could you also remove `.github/ISSUE_TEMPLATE/question.yml` before merging since now it's empty?

**Comment by leonsteinhaeuser** [comment]
> great! maybe could you also remove `.github/ISSUE_TEMPLATE/question.yml` before merging since now it's empty?

That makes sense. I used the GitHub merge buttons and it seems that one of the changes did not delete the question template. I have now done this by hand.

**Comment by scarf005** [comment]
> At least 1 approving review is required by reviewers with write access. 

since my review approval doesn't count (no write access), you could safely merge with administer rights .

**Pull request merged**